lib/perl/Genome/Model/ClinSeq.pm

```diff
@@ -966,6 +810,10 @@ sub _resolve_workflow_for_build {
                     destination_property => 'converge_snv_indel_report_result',
                 );
             }
+            if (($mqs->[$i-1] + $bqs->[$i-1]) > $best_mq_bq_sum) {
+                $best_mq_bq_sum = $mqs->[$i-1] + $bqs->[$i-1];
```

Comment:
Would it be possible to change the range of `$i` to `(0..$#$mqs)` to avoid these `-1`s?  Or maybe a variable that holds `$i-1` to give meaning to the repeated math?